packages/remix-cloudflare/globals.ts

```diff
@@ -16,3 +16,5 @@ export function installGlobals() {
   self.sign = sign;
   self.unsign = unsign;
 }
+
+installGlobals();
```

Comment:
We shouldn't need to install any globals on Cloudflare.

Reply:
Once we remove the `sign`/`unsign` globals, will we still need to declare types for `process`/`NODE_ENV` for Cloudflare?

Reply:
We do currently encourage the use of `process.env.NODE_ENV` in user code to distinguish environment, so yeah. We still need that one.

Reply:
Note: Someday we should support a `REMIX_BUILD_MODE` global or something like that to replace `process.env.NODE_ENV`, so it feels more natural on non-node environments.

Reply:
Since we no longer `import "./globals"`, do we need to do anything to get the globals.d.ts file in  remix-cloudflare to be used by Remix apps that depend on remix-cloudflare? Do we need to point the app's tsconfig to that globals.d.ts file?